### PR TITLE
Update security considerations

### PR DESCRIPTION
**Pull request by gkellogg**
and add privacy and internationalization considerations appendices.

For #147, and #148.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12220,14 +12228,47 @@ <h3>Examples</h3>
...
 <section id="security" class="appendix">
   <h3>Security Considerations</h3>
-  <p class="ednote">Consider requirements from <a href="https://w3ctag.github.io/security-questionnaire/">Self-Review Questionnaire: Security and Privacy</a>.</p>
-  <p>See, <a href="#iana-considerations" class="sectionRef"></a></p>
+  <p>See, <a href="#iana-security">Security Considerations</a> in <a href="#iana-considerations" class="sectionRef"></a>.</p>
+</section>
+
+<section id="privacy" class="appendix">
+  <h3>Privacy Considerations</h3>
+  <p>The retrieval of external contexts can expose operation of JSON-LD processor
```
Comment (gkellogg) [inline index.html]:
```suggestion
  <p>The retrieval of external contexts can expose the operation of a JSON-LD processor
```

File: index.html [hunk 2]
```diff
@@ -12220,14 +12228,47 @@ <h3>Examples</h3>
 
 <section id="security" class="appendix">
   <h3>Security Considerations</h3>
-  <p class="ednote">Consider requirements from <a href="https://w3ctag.github.io/security-questionnaire/">Self-Review Questionnaire: Security and Privacy</a>.</p>
-  <p>See, <a href="#iana-considerations" class="sectionRef"></a></p>
+  <p>See, <a href="#iana-security">Security Considerations</a> in <a href="#iana-considerations" class="sectionRef"></a>.</p>
```
Comment (iherman) [inline index.html]: We did say on the call that adding some paragraph on the future need for context as well as JSON data file signature might be a good idea. Should it be here?

Comment (iherman) [reply index.html]: Do we want to add a reference to issue #86 ?

[the remark this answers is not in the document]
Comment (gkellogg) [reply index.html]: We do in the privacy section:

> Whenever practical, consider the use of a secure connection, such as HTTPS when referencing external contexts.

[the remark this answers is not in the document]
Comment (iherman) [reply index.html]: Ah! I missed that.

File: index.html [hunk 3]
```diff
@@ -12220,14 +12228,55 @@ <h3>Examples</h3>
...
```
Comment (iherman) [inline index.html]: (I have not checked before, sorry): I guess these sections (security, privacy, etc) should not be appendices, but are supposed to be sections in the final text...

File: index.html [hunk 4]
```diff
@@ -12220,14 +12228,55 @@ <h3>Examples</h3>
...
+  <p>Future versions of this specification
+    may incorporate subresource integrity [[?SRI]] as a means of ensuring that cached and retrieved
+    content matches data retrieved from remote servers; see <a href="https://github.com/w3c/json-ld-syntax/issues/86">issue 86</a>.</p>
+</section>
```
Comment (iherman) [inline index.html]: I think this last paragraph should be of class note; it is definitely a non-normative statement.

**Pull request merged**